From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Rec <recrumptor@gmail.com>
Date: Fri, 10 Oct 2025 20:36:59 +0500
Subject: [PATCH 4/7] Update mt7621.mk

Signed-off-by: Rec <recrumptor@gmail.com>
---
 target/linux/ramips/image/mt7621.mk | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/target/linux/ramips/image/mt7621.mk b/target/linux/ramips/image/mt7621.mk
--- a/target/linux/ramips/image/mt7621.mk
+++ b/target/linux/ramips/image/mt7621.mk
@@ -2594,6 +2594,30 @@ define Device/netis_wf2881
endef
TARGET_DEVICES += netis_wf2881

define Device/notion_r2x1
  $(Device/nand)
  $(Device/uimage-lzma-loader)
  IMAGE_SIZE := 128768k
  IMAGES += factory.bin
  IMAGE/factory.bin := append-kernel | pad-to $$$$(KERNEL_SIZE) | append-ubi | \
	check-size
  DEVICE_VENDOR := Notion
  DEVICE_PACKAGES := kmod-mt7603 kmod-mt7615e kmod-mt7663-firmware-ap \
	kmod-usb3 kmod-usb-net-rndis comgt-ncm -uboot-envtools
endef

define Device/notion_r281
  $(Device/notion_r2x1)
  DEVICE_MODEL := R281
endef
TARGET_DEVICES += notion_r281

define Device/notion_r291
  $(Device/notion_r2x1)
  DEVICE_MODEL := R291
endef
TARGET_DEVICES += notion_r291

define Device/openfi_5pro
  $(Device/dsa-migration)
  IMAGE_SIZE := 65216k
